src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
They aren't just antisocial- they're a little flaky as they're going to be picked up by the AV (and locked). Popping notepad isn't quite as bad as (accidentally) deleting user documents- while putting some in OuterLoop will mitigate that, it still makes me uneasy.

But again, I'm not done here.